src/library-authoring/components/ComponentCard.tsx

```diff
@@ -84,15 +89,18 @@ const ComponentCard = ({ contentHit, blockTypeDisplayName } : ComponentCardProps
 
   return (
     <Container className="library-component-card">
-      <Card>
+      <Card
+        isClickable
+        onClick={() => openComponentInfoSidebar(usageKey)}
+      >
```

Comment:
Although the cards can be focused using keyboard, [pressing enter or space](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard?utm_source=devtools&utm_medium=a11y-panel-checks-keyboard#focusable_elements_should_have_interactive_semantics) key doesn't trigger click event so it is not really accessible. We probably need to add a keydown handler which triggers click event.

Reply:
Good catch! Fixed here: b2a05d53ee2d80642420c5b7b799ffed3a9f9bf4